streams/src/test/java/org/apache/kafka/streams/TopologyTest.java

```diff
@@ -330,8 +327,6 @@ public void shouldNotAllowToAddStateStoreToSink() {
 
     private void mockStoreBuilder() {
         when(storeBuilder.name()).thenReturn("store");
-        when(storeBuilder.logConfig()).thenReturn(Collections.emptyMap());
```

Comment:
I assume Mockito said these are unnecessary?

Reply:
Yeah, it is somehow strange. We tried to define Mockito with strict stubs but now with the migration to JUnit 5 it turns out that it was that strict after all. I haven't yet understood why we get unnecessary stubbing exceptions. 🤔 
In this specific case, it does not make any difference from a stubbing point of view since the return values are the default return values. So instead of considering `lenient`, I just removed the stubs.